python/vyos/configdict.py

```diff
@@ -119,6 +119,8 @@ def vlan_to_dict(conf):
         'disable_link_detect': 1,
         'egress_qos': '',
         'egress_qos_changed': False,
+        'id': '',
```

Comment:
This is not shared by all the code, this is why when I moved this "vlan" (really I should have renamed this ! I did not add it. I believe that adding it, would set the expectation that intf_ code would set it.